6_visualize.yml

```diff
@@ -22,13 +22,14 @@ sources:
   - 6_visualize/src/prep_precip_fun.R
   - 6_visualize/src/prep_rivers_fun.R
   - 6_visualize/src/prep_legend_fun.R
+  - 6_visualize/src/prep_datetime_fun.R
```

Comment:
It would be slightly more precise if you declared this as a source in `create_storm_gif_makefile` rather than in 6_visualize.yml, because the the targets that will need this script are in 6_storm_gif_tasks.yml.

In the branch I'm working on right now, I've moved around a couple other functions with this in mind, because prep_storm_point_fun and prep_precip_fun both also fall into this category. Right now I have:
```r
create_storm_gif_makefile <- function(makefile, task_plan, remake_file) {
  scipiper::create_task_makefile(
    makefile=makefile, task_plan=task_plan,
    include=remake_file,
    packages=c('dplyr', 'scipiper'),
    sources=c(
      '6_visualize/src/prep_storm_point_fun.R',
      '6_visualize/src/prep_precip_fun.R',
      '6_visualize/src/create_animation_frame.R'),
    file_extensions=c('feather','ind'),
    ind_complete=TRUE)
}
```
so you could add this line as one more source in that function rather than here. I'll probably get a small merge conflict after you've done this, once I get back to working on my branch, but should be easy for me to resolve.

Reply:
ah yup I'll put it in that yaml